src/ViewerComponents/ResourceViewer.svelte

```diff
@@ -0,0 +1,415 @@
+                                label: ontology['rdfs:label'],
+                                customName: cName ? cName : null
+                            };
+                        }
+                    }
+                })
+                break;
+            case 'knora-api:ListValue':
+                const listObject = await listNodeRequest(propValue['knora-api:listValueAsListNode']['@id']);
+                // console.log(listObject, listObject['knora-api:hasRootNode']['@id']);
+                $lists.forEach(list => {
+                    if (list.id === listObject['knora-api:hasRootNode']['@id']) {
+                        // TODO There are sometimes more than one label
+                        if (properties[listObject['knora-api:hasRootNode']['@id']]) {
+                            properties[listObject['knora-api:hasRootNode']['@id']]['values'].push(listObject['rdfs:label']);
+                        } else {
+                            properties[listObject['knora-api:hasRootNode']['@id']] = {
+                                values: new Array(listObject['rdfs:label']),
+                                label: list['labels'][0]['value'],
+                                customName: cName ? cName : null
+                            };
+                        }
+                    }
+                })
+                break;
+            case 'knora-api:LinkValue':
+                if (customProp) {
+                    if (!customProp['linkResource'] ) {
+                        return;
+                    }
+
+                    const reqLinkTarget = await linkRequest(propValue['knora-api:linkValueHasTarget']['@id'])
+                    processCustomProp(reqLinkTarget, customProp['linkResource']['Props']);
+
+                } else {
+                    $ontologies.forEach(ontology => {
+                        if (ontology['@id'] === pName) {
+                            if (properties[pName]) {
+                                properties[pName]['values'].push(propValue['knora-api:linkValueHasTarget']['@id']);
+                            } else {
+                                properties[pName] = {
+                                    values: new Array(propValue['knora-api:linkValueHasTarget']['@id']),
+                                    label: ontology['rdfs:label'],
+                                    customName: cName ? cName : null
+                                };
+                            }
+                        }
+                    })
+                }
+                break;
+            default:
+                break;
+        }
+        console.log(properties);
+    }
+
+</script>
+
+<main>
+    <h2>Resource Viewer</h2>
+    {#if error}
+        <div>
+            There was a error! The Resource data couldn't be loaded.
+            <button on:click={() => login()}>Try again</button>
+        </div>
+    {:else}
+        <div>
+            <button on:click={() => login()}>Get Data</button>
```

Comment:
Same as on the image viewer, would be nice if the data was loaded without a click

Reply:
I agree